src/blueprints/timeline-state-resolver-types/sisyfos-types.ts

```diff
@@ -21,6 +21,15 @@ export interface SisyfosChannelsTimelineObject extends TimelineObject {
   }
 }
 
+// Todo: Figure out if 'resync' can be changed to `resynchronize`, or if it is the value that Sisyfos expects.
+export interface SisyfosResynchronizeTimelineObject extends TimelineObject {
```

Comment:
Might help if i place it at the correct line. It was aimed at the attribute of the content. Has been moved to just above that one.

Reply:
With that said, is the name of the attributes in `content`, set in stone, or can they be changed?

Reply:
> Might help if i place it at the correct line.

Perfect example as to why we want to minimize comments because they become outdated/moved from what they are commenting on extremely quickly. 
This one didn't even last the commit it was introduced in...

Reply:
Todo has been removed.